### ISSUE TITLE
[Bug]: CLI Command no longer able to import translations from file with \Pimcore\Model\Translation::importTranslationsFromFile

### ISSUE THREAD

**Issue by Fahl-Design** [pimcore/pimcore#12910]
### Expected behavior

I can use `\Pimcore\Model\Translation::importTranslationsFromFile` to import a CSV file with a custom command like

```php
            \Pimcore\Model\Translation::importTranslationsFromFile(
                $adminTranslationFilePath,
                \Pimcore\Model\Translation::DOMAIN_ADMIN,
                $input->getOption('replaceExistingTranslation')
            );
```

### Actual behavior

getting an error and import skipped:
`11:08:16 WARNING   [pimcore] User system not allowed to edit 0.000000e+0 translation` from
`project/vendor/pimcore/pimcore/models/Translation/Dao.php:96`
and by the way the in the `sprintf` call `%e` must be replaces with `%s` anyway:
```
'User %s not allowed to edit %e translation'
```

### Steps to reproduce

1) create an CSV file like:
```
key;de;en;fr;it;nl;pl
object.marketing_rule.error.invalid_bonus_quantity;Die Menge des Bonus ist ungültig;"";"";"";"";""
```

2) create a simple command to execute:

```php
            \Pimcore\Model\Translation::importTranslationsFromFile(
                $adminTranslationFilePath,
                \Pimcore\Model\Translation::DOMAIN_ADMIN,
                true
            );
```

3) an see errors and no translation is inserted/updated

**Comment by dvesh3** [comment]
Regression of #12756

**Comment by aryaantony92** [comment]
Fixed by #12964
